apps/web/src/pages/Pools/PoolAdvertisementPage/components/ApplicationLink.tsx

```diff
@@ -33,29 +35,33 @@ const ApplicationLink = ({
   const href = applicationId
     ? paths.application(applicationId)
     : paths.createApplication(poolId);
-
-  let linkText = intl.formatMessage({
-    defaultMessage: "Apply for this process",
-    id: "W2YIEA",
-    description: "Link text to apply for a pool advertisement",
-  });
-  if (applicationId) {
-    linkText = hasApplied
-      ? intl.formatMessage({
-          defaultMessage: "View my application",
-          id: "btCYxZ",
-          description: "Link text to view an existing, submitted application",
-        })
-      : intl.formatMessage({
-          defaultMessage: "Continue my application",
-          id: "g5JeNf",
-          description: "Link text to continue an existing, draft application",
-        });
+  let linkTextLabel;
+  if (!linkText) {
+    linkTextLabel = intl.formatMessage({
+      defaultMessage: "Apply for this process ",
```

Comment:
Remove trailing space that was removed (the string and its French translation already existed, `W2YIEA`)
```suggestion
      defaultMessage: "Apply for this process",
```